scripts/docs/snippetadapter.mjs

```diff
@@ -136,12 +136,31 @@ async function buildSnippets( snippets, paths, constants, imports ) {
 			 * bundled. This will cause issues, because the `tests` directory is not available in the CDN build.
 			 */
 			{
-				name: 'external',
+				name: 'externalize-ckeditor5',
 				setup( build ) {
 					build.onResolve( { filter: /.*/ }, args => ( {
 						external: externals.some( name => name.endsWith( '/' ) ? args.path.startsWith( name ) : args.path === name )
 					} ) );
 				}
+			},
+
+			/**
+			 * Code from "ckeditor5" and "ckeditor5-premium-features" is externalized to avoid bundling it with the snippet
+			 * code. However, if the source file is imported directly, it can be bundled and (in case of commercial features)
+			 * not obfuscated. To prevent this, we throw an error if such an import is detected.
+			 */
+			{
+				name: 'throw-on-source-import',
+				setup( build ) {
+					build.onLoad( { filter: /packages\/(.*)\/src/ }, args => ( {
```

Comment:
It seems that paths have OS-specific separators, because currently this filter does not match anything on Windows.

```suggestion
					build.onLoad( { filter: /packages[\\/](.*)[\\/]src/ }, args => ( {
```

Reply:
```suggestion
					build.onLoad( { filter: /packages[\\/].*[\\/]src/ }, args => ( {
```

We do not have to capture anything. I am unsure if it speeds up (:D).

Reply:
Fixed